python/cudf/cudf/tests/test_factorize.py

```diff
@@ -25,7 +25,26 @@ def test_factorize(ncats, nelem):
     np.testing.assert_array_equal(uvals.to_array(), handcoded)
 
 
-def test_factorize_index():
+@pytest.mark.parametrize("ncats,nelem", [(2, 2), (2, 10), (10, 100)])
+def test_factorize_index_obj(ncats, nelem):
```

Comment:
Can we add a test of using `cudf.factorize` against a cupy array?

Reply:
Done.